renderer/components/Login/Login.js

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
+import { FormattedMessage } from 'react-intl'
+import { Box, Flex } from 'rebass/styled-components'
+import { Button, CenteredContent, Message, Text } from 'components/UI'
+import { Form, PasswordInput } from 'components/Form'
+import ArrowRight from 'components/Icon/ArrowRight'
+import ZapLogo from 'components/Icon/ZapLogo'
+import messages from './messages'
+
+const Login = ({ login, loginError, clearLoginError, ...rest }) => {
+  const formApiRef = useRef(null)
+
+  useEffect(() => {
+    const { current: formApi } = formApiRef
+    loginError && formApi.setFormError(loginError)
```

Comment:
if `loginError` is not set, do we need to call clear ?
maybe frame this two into `if (loginError)...`